tests/test_icml_conference.py

```diff
@@ -4620,6 +4618,10 @@ def test_post_decision_stage(self, client, openreview_client, helpers):
 
         assert '_bibtex' in rejected_submissions[0].content and rejected_submissions[0].content['_bibtex']['value'] == valid_bibtex
 
+        #make sure all decision process functions have finished
+        for number in range(1, 101):
+            helpers.await_queue_edit(openreview_client, invitation=f'ICML.cc/2023/Conference/Submission{number}/-/Decision')
```

Comment:
The function has a count that is 1 by default, you can pass the count instead of calling the same functions 100 times.

Reply:
Disregard my comment, I see it is a different invitation per note.